src/db/tables/subscribers.js

```diff
@@ -124,6 +124,18 @@ async function removeFxAData (subscriber) {
   return updatedSubscriber
 }
 
+/**
+ * @param {import('./subscribers_types').SubscriberRow} subscriber
+ * @param {number} onerepProfileId
+ */
+async function setOnerepProfileId (subscriber, onerepProfileId) {
+  await knex('subscribers')
+    .where('id', '=', subscriber.id)
```

Comment:
Nit: KNEX also [supports](https://knexjs.org/guide/query-builder.html#where):
```suggestion
    .where('id', subscriber.id)
```

Reply:
Ah, good to know. I hadn't used Knex before, so I think I just got this from the docs: 97139c2ea